server/repositories/CaptureRepository.js

```diff
@@ -2,8 +2,8 @@ const BaseRepository = require('./BaseRepository');
 
 class CaptureRepository extends BaseRepository {
   constructor(session) {
-    super('capture_denormalized', session);
-    this._tableName = 'capture_denormalized';
+    super('reporting.capture_denormalized', session);
```

Comment:
You should remove the 'reporting.'. In the cloud, the DATABASE_URL is already configured only to access the reporting schema, so this is unnecessary. This might also be the reason for the failed tests

Reply:
Ah, my bad. I thought I committed the code without the `reporting.` prefix.